packages/google_maps_flutter/google_maps_flutter_web/lib/src/marker_clustering_js_interop.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(srujzs): Needed for https://github.com/dart-lang/sdk/issues/54801. Once
+// we publish a version with a min SDK constraint that contains this fix,
+// remove.
+@JS()
+library;
```

Comment:
@ditman @srujzs 
Should I remove these lines and update the SDK constraints to 3.3.1 within this PR, or should this be updated separately?
See: https://github.com/CodemateLtd/packages/pull/6#discussion_r1581648828

Reply:
I think we can clean them up all together later.